workers/data_refinery_workers/downloaders/utils.py

```diff
@@ -109,17 +109,22 @@ def start_job(job_id: int, max_downloader_jobs_per_node=MAX_DOWNLOADER_JOBS_PER_
         logger.error("This downloader job has already been started!!!", downloader_job=job.id)
         raise Exception("downloaders.start_job called on a job that has already been started!")
 
-    for original_file in job.original_files.all():
-        if original_file.has_ever_been_processed():
-            logger.error(("Original file has a successful processor job, it doesn't need"
-                          " to be downloaded! Aborting!"),
-                         job_id=job.id,
-                         original_file_id=original_file.id
-            )
-            job_context["original_files"] = []
-            job_context["computed_files"] = []
-            job_context['abort'] = True
-            return job_context
+    # Only do this for SRA jobs because they don't have archives which
+    # this isn't capable of dealing with yet.
+    if job.downloader_task == "SRA":
+        # SRA jobs only have one sample.
+        sample = job.orignal_files.first().samples().first():
+        for computed_file in sample.computed_files.all():
+            if compted_file.s3_bucket and compted_file.s3_key:
+                logger.error(("Sample has a good computed file, it must have been processed, "
+                              "so it doesn't need to be downloaded! Aborting!"),
+                             job_id=job.id,
+                             original_file_id=original_file.id
+                )
+                job_context["original_files"] = []
```

Comment:
start_job does not have a job_context. The code above uses `sys.exit(0)` after setting job.success explicitly.